compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -12,11 +12,42 @@ import config.Config
 import config.Printers.lexical
 import config.Settings.Setting
 import Tokens._
-import scala.annotation.{ switch, tailrec }
+import scala.annotation.{switch, tailrec}
 import scala.collection.mutable
 import scala.collection.immutable.{SortedMap, BitSet}
 import rewrites.Rewrites.patch
 
+object Cbufs {
+  import java.lang.StringBuilder
+
+  private final val TargetCapacity = 256
+
+  opaque type Cbuf = StringBuilder
+  object Cbuf:
+    def apply(): Cbuf = new StringBuilder(TargetCapacity)
+
+  extension StringBuilderOps on (buf: Cbuf):
```

Comment:
Just wondering: Since you used indentation here, why not for the other parts?

Reply:
Learning curve. First time using any of this syntax! Most surprising was a strange compulsion to use `end if`; maybe that will pass.

Reply:
You can if you want to 😉